### DISCUSSION TITLE
Realtime Database Web

### DISCUSSION THREAD

**Discussion by nanashili** [firebase/flutterfire#3858] [Feature request]
How long will we still have to wait for realtime database for flutter web.

**Comment by dukenebur** [comment]
Please, we really need that

**Comment by alejandroaap** [comment]
I still don't understand why flutter plugins are way behind the native versions. Is there an explanation?

**Reply by yusuf335** [reply]
Yes, it will be really great to have a realtime database with the web. As an electrical engineering student now most of our project is IoT (Internet of things) base. Recently I build a water monitor system with the realtime database but the lacking is the web admin panel to monitor it. I will request and hope that the developer and contributor will look on this and release an update for the web.

**Comment by khp53** [comment]
We really need the flutter web for firebase real time database as soon as possible.

**Comment by edsson-madrigal** [comment]
Please Flutter, it would be wonderful if you could add real-time database

**Comment by devDeejay** [comment]
We need Realtime Database Support for Flutter Web. Cloud firestore might not be the way to go for all the Flutter web apps. Plus Flutter is from Google we expect good integrations with all Google services atleast.
